File: tests/test-minecraft/src/main/java/pize/tests/minecraft/game/audio/Sound.java
package pize.tests.minecraft.game.audio;

import pize.tests.minecraft.game.options.SoundCategory;

public enum Sound{

    CLICK(SoundCategory.MASTER, "click", 0.25F);


    public final SoundCategory category;
    public final String resourceId;
    public final float maxVolume;

    Sound(SoundCategory category, String resourceId, float maxVolume){
        this.category = category;
        this.resourceId = resourceId;
        this.maxVolume = maxVolume;
    }

}

File: tests/test-minecraft/src/main/kotlin/pize/tests/minecraft/game/audio/Sound.kt
package pize.tests.minecraft.game.audio

import pize.tests.minecraft.game.options.SoundCategory

enum class Sound private constructor(val category: SoundCategory, val resourceId: String, val maxVolume: Float) {
    CLICK(SoundCategory.MASTER, "click", 0.25f)
}
